script/review-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -e
+yarn install --production=false
+npm run build -- --buildtype localhost --api='${API_URL}'
+npm run heroku-serve -- build/localhost -p 3001
```

Comment:
If there's a way we can run a dev server that listens on more than just 127.0.0.1, I'd love to switch if we could get auto-reloading.